### PR TITLE
Update Yuri.

### PR DESCRIPTION
**Pull request by GraionDilach**
Refer to individual commit messages what are changed.

### CODE REVIEW

**Comment by GraionDilach** [comment]
Apparently this is crashing atm so don't merge it just yet.

File: OpenRA.Mods.RA2/Traits/Render/WithMindControlArc.cs [hunk 1]
```diff
@@ -30,11 +30,17 @@ public class WithMindControlArcInfo : ITraitInfo
...
 
-		[Desc("Draw with this many piecewise-linear lines")]
-		public readonly int Segments = 16;
+		[Desc("Controls how fine-grained the resulting arc should be.")]
+		public readonly int QuantizedSegments = 16;
+
+		[Desc("Equivalent to sequence ZOffset. Controls Z sorting.")]
+		public readonly int ZOffset = 0;
+
+		[Desc("The width of the zap.")]
```
Comment (GraionDilach) [inline OpenRA.Mods.RA2/Traits/Render/WithMindControlArc.cs]: NTM: facepalm.

File: mods/ra2/sequences/defaults.yaml [hunk 2]
```diff
@@ -75,3 +75,9 @@
 		Start: 276
 	cheer:
 		Start: 293
+
+^MindControllable:
+	mindcontrol: yuricntl
+		Length: *
+		Offset: 0, -24, 24
+
```
Comment (GraionDilach) [inline mods/ra2/sequences/defaults.yaml]: Whoops.

**Comment by GraionDilach** [comment]
Rebased & fixed.

File: OpenRA.Mods.RA2/Traits/Render/WithMindControlArc.cs [hunk 3]
```diff
@@ -18,23 +18,31 @@
 
 namespace OpenRA.Mods.RA2.Traits
 {
+	[Desc("Draws an arc between a mindcontroller actor and all its victims",
+		"or a actively-mindcontrolled actor and it's controller.")]
```
Comment (abcdefg30) [inline OpenRA.Mods.RA2/Traits/Render/WithMindControlArc.cs]: `an actively mindcontrolled actor`

**Comment by GraionDilach** [comment]
Fixed.

**Comment by DylanC** [comment]
@GraionDilach @abcdefg30 - Ready to merge? :)

**Comment by GraionDilach** [comment]
I don't have merge rights but AFAIK it is.

**Comment by Mailaender** [comment]
I get the following crash when starting the map Dustbowl:

```
Exception of type `System.InvalidOperationException`: Unit `suvw` does not have any sequences defined.
  at OpenRA.Graphics.SequenceProvider.GetSequence (System.String unitName, System.String sequenceName) [0x0002d] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Graphics.Animation.GetSequence (System.String sequenceName) [0x0000d] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Graphics.Animation.PlaySequence (System.String sequenceName) [0x00001] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Graphics.Animation.PlayRepeating (System.String sequenceName) [0x0000f] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Mods.Common.Traits.Render.WithIdleOverlay..ctor (OpenRA.Actor self, OpenRA.Mods.Common.Traits.Render.WithIdleOverlayInfo info) [0x00100] in <da850878f47b4c469cfe91761a3864ce>:0 
  at OpenRA.Mods.Common.Traits.Render.WithIdleOverlayInfo.Create (OpenRA.ActorInitializer init) [0x00001] in <da850878f47b4c469cfe91761a3864ce>:0 
  at OpenRA.Actor..ctor (OpenRA.World world, System.String name, OpenRA.Primitives.TypeDictionary initDict) [0x000af] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.World.CreateActor (System.Boolean addToWorld, System.String name, OpenRA.Primitives.TypeDictionary initDict) [0x00001] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.World.CreateActor (System.String name, OpenRA.Primitives.TypeDictionary initDict) [0x00001] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Mods.Common.Traits.SpawnMapActors.WorldLoaded (OpenRA.World world, OpenRA.Graphics.WorldRenderer wr) [0x000a0] in <da850878f47b4c469cfe91761a3864ce>:0 
  at OpenRA.World.LoadComplete (OpenRA.Graphics.WorldRenderer wr) [0x00080] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Game.StartGame (System.String mapUID, OpenRA.WorldType type) [0x000c3] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Network.UnitOrders.ProcessOrder (OpenRA.Network.OrderManager orderManager, OpenRA.World world, System.Int32 clientId, OpenRA.Order order) [0x005b9] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Network.OrderManager.TickImmediate () [0x00109] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Sync+<CheckSyncUnchanged>c__AnonStorey0.<>m__0 () [0x00001] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Sync.CheckSyncUnchanged[T] (OpenRA.World world, System.Func`1[TResult] fn) [0x00007] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Sync.CheckSyncUnchanged (OpenRA.World world, System.Action fn) [0x0000e] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Game.InnerLogicTick (OpenRA.Network.OrderManager orderManager) [0x0010a] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Game.LogicTick () [0x00050] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Game.Loop () [0x000d6] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Game.Run () [0x00042] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Game.InitializeAndRun (System.String[] args) [0x00011] in <d64e6744391b472fa46cb62aa279c746>:0 
  at OpenRA.Program.Main (System.String[] args) [0x0004f] in <d64e6744391b472fa46cb62aa279c746>:0
```

**Pull request merged**